### PR TITLE
Feature/migrate dotnet 2.2 to 3.1

### PR DESCRIPTION
**Pull request by jcii**

### CODE REVIEW

File: Machete.Web/Program.cs [hunk 1]
```diff
@@ -54,18 +86,18 @@ public static async Task Main(string[] args)
...
+                //     logging.AddConfiguration(app.Configuration.GetSection("Logging"));
+                //     logging.AddConsole();
+                //     logging.AddDebug();
+                //     logging.AddEventSourceLogger();
+                //     logging.AddFilter("Microsoft.EntityFrameworkCore.Database.Command", LogLevel.Warning);
+                //     logging.AddFilter("Microsoft.EntityFrameworkCore.Infrastructure", LogLevel.Warning);
+                //
+                // })
+                // .UseStartup<Startup>()
+                // .UseUrls("http://*:4213");
```
Comment (chaim1221) [inline Machete.Web/Program.cs:101]: the comments can be deleted, although I don't know why we're switching out WebHostBuilder for HostBuilder, so maybe a comment to that effect

File: appveyor_install.sh [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
-git submodule update
-npm install -g @angular/cli
-docker run -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=passw0rD!' --network machete-bridge -p 1433:1433 --name sqlserver -d mcr.microsoft.com/mssql/server
+#git submodule init
+#git submodule update
+#npm install @angular/cli
+dotnet restore
+docker ps -a
+docker network ls
+#docker run -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=passw0rD!' --network machete-bridge -p 1433:1433 --name sqlserver -d mcr.microsoft.com/mssql/server
```
Comment (chaim1221) [inline appveyor_install.sh:10]: this can probably be renamed

File: global.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "version": "3.1.102",
+    "rollForward": "latestFeature"
```
Comment (chaim1221) [inline global.json:4]: :+1:

File: Machete.Data/Initialize/MacheteLookups.cs [hunk 4]
```diff
@@ -208,7 +208,7 @@ public static class MacheteLookups
                 context.Lookups.Add(u);
             });
             context.Database.OpenConnection();
-            context.Database.ExecuteSqlCommand("SET IDENTITY_INSERT dbo.Lookups ON");
+            context.Database.ExecuteSqlRaw("SET IDENTITY_INSERT dbo.Lookups ON");
```
[the remark this answers is not in the document]
Comment (jcii) [reply Machete.Data/Initialize/MacheteLookups.cs:211]: Command deprecated

[the remark this answers is not in the document]
Comment (jcii) [reply Machete.Data/Initialize/MacheteLookups.cs:211]: Initialize/MacheteTransports.cs(14,17): warning CS0618: 'RelationalDatabaseFacadeExtensions.ExecuteSqlCommand(DatabaseFacade, RawSqlString, params object[])' is obsolete: 'For the execution of SQL queries using plain strings, use ExecuteSqlRaw instead. For the execution of SQL queries using interpolated string syntax to create parameters, use ExecuteSqlInterpolated instead.' [/home/jimmy/src/Machete/Machete.Data/Machete.Data.csproj]

**Review by chaim1221 — APPROVED** [review]

**Pull request merged**